internal/controllers/vgmanager/kname_test.go

```diff
@@ -1,17 +1,18 @@
 package vgmanager
 
 import (
+	"github.com/openshift/lvm-operator/v4/api/v1alpha1"
```

Comment:
Same here. 

We try to keep a consistent import order throughout the codebase:
- golang imports
- github imports
- k8s imports
(with new lines in between)